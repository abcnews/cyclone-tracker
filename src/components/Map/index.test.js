const React = require('react');
const renderer = require('react-test-renderer');

const Map = require('../Map');

describe('Map', () => {
  test('It renders', () => {
    const component = renderer.create(<Map />);

    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
