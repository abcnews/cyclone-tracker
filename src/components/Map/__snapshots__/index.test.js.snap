// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Map It renders 1`] = `
<div
  className="base"
/>
`;
